internal/cmd/wave/create.go

```diff
@@ -531,3 +622,40 @@ func allNonDeletedStackStatuses() []string {
 func defaultStackStatusFilter() []*string {
 	return aws.StringSlice(allNonDeletedStackStatuses())
 }
+
+func updateOcean(ctx context.Context, logger logr.Logger) error {
+
+	conf, err := config.GetConfig()
+	if err != nil {
+		return fmt.Errorf("cannot get cluster configuration, %w", err)
+	}
+
+	const oceanURL = "https://s3.amazonaws.com/spotinst-public/integrations/kubernetes/cluster-controller/spotinst-kubernetes-cluster-controller-ga.yaml"
+
+	res, err := http.Get(oceanURL)
+	if err != nil {
+		return fmt.Errorf("error fetching ocean manifests, %w", err)
+	}
+	data, err := ioutil.ReadAll(res.Body)
+	defer res.Body.Close()
+	if err != nil {
+		return fmt.Errorf("error reading ocean manifests, %w", err)
+	}
+
+	delim := regexp.MustCompile("(?m)^---$")
+	objects := delim.Split(string(data), -1)
```

Comment:
not sure why the yaml libraries don't split on "---" according to the spec, but that's where we are